### PR TITLE
Makes injuries more gruesome

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: Tacolizard Forever: mangled and bruised!
add: Examining people will now reveal much more specific and gruesome details about their injuries.
/:cl:

[why]: I've been playing DF a lot and really enjoyed looking at the detailed injury information on my dwarves. I have changed our injury messages to describe more visible damage, and less 'he has severe burns' which is just bland and boring.

~~I may get around to giving people 'scars' after they are healed and mabye add different injury info depending on whether someone was beaten with a blunt object or cut with something sharp.~~

Github Desktop (i don't know why i use it oh god) has borked everything again so I'm going to delete my fork and start over if/when this gets merged then make any changes.

### CODE REVIEW

File: code/modules/mob/living/carbon/examine.dm [hunk 1]
```diff
@@ -45,26 +45,50 @@
...
-			msg += "[t_He] [t_has] minor burns.\n"
+			if(prob(50))
+				msg += "[t_his] skin is bright red.\n"
+			else
+				msg += "[t_his] skin is covered in small burns.\n"
 		else
-			msg += "<B>[t_He] [t_has] severe burns!</B>\n"
+			if(prob(33))
+				msg += "<B>[t_his] skin is covered in large blisters!</B>\n"
+			else if(prob(33))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/examine.dm]: prob(50)

File: code/modules/mob/living/carbon/examine.dm [hunk 2]
```diff
@@ -45,26 +45,50 @@
...
-			msg += "[t_He] [t_has] minor bruising.\n"
+			if(prob(50)
+				msg += "[t_He] [t_has] small bruises.\n"
+			else
+				msg += "[t_He] [t_has] minor bruising.\n"
 		else
-			msg += "<B>[t_He] [t_has] severe bruising!</B>\n"
+			if(prob(33))
+				msg += "<B>[t_He] [t_is] bloodied!</B>\n"
+			else if(prob(33))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/examine.dm]: prob(50)

File: code/modules/mob/living/carbon/examine.dm [hunk 3]
```diff
@@ -45,26 +45,50 @@
...
-			msg += "[t_He] [t_is] slightly deformed.\n"
+			if(prob(50))
+				msg += "[t_He] [t_is] covered in minor deformities.\n"
+			else
+				msg += "[t_his] body is deformed.\n"
 		else
-			msg += "<b>[t_He] [t_is] severely deformed.</b>\n"
+			if(prob(33))
+				msg += "<B>[t_his] skin is covered in large, unnatural genetic deformities!</B>\n"
+			else if(prob(33))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/examine.dm]: prob(50)

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Change the statistics a bit to get what you actually want

**Comment by Tacolizard** [comment]
locally I'm getting over 350 compile errors from saycode about empty 'else' statements. I have no idea what is going on

File: code/modules/mob/living/carbon/examine.dm [hunk 4]
```diff
@@ -50,9 +50,9 @@
 			else
 				msg += "[t_He] [t_has] minor bruising.\n"
 		else
-			if(prob(33))
+			if(prob(50))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/examine.dm]: I mean the first if is prob(33) the second should be prob(50)

Think of it this way, you start with 100 cases

You take 33% of that, you now have 66 cases

Take another 33% of that (22) and you'll have 44 cases

Taking 50% of the 66 cases is what you want in the second if

TL;DR

if(prob(50))
   //50% chance
else if(prob(50))
   //25% chance
else
   //25% chance

if(prob(33))
  //33% chance
else if(prob(33))
  //22% chance
else
  //44% chance

if(prob(33))
  //33% chance
else if(prob(50)
  //33% chance
else
  //33% chance

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Tacolizard** [comment]
nevermind im stupid

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Reposting cause I got outdated

I meant the first if is prob(33) the second should be prob(50)

Think of it this way, you start with 100 cases

You take 33% of that, you now have 66 cases

Take another 33% of that (22) and you'll have 44 cases

Taking 50% of the 66 cases is what you want in the second if

TL;DR

if(prob(50))
//50% chance
else if(prob(50))
//25% chance
else
//25% chance

if(prob(33))
//33% chance
else if(prob(33))
//22% chance
else
//44% chance

if(prob(33))
//33% chance
else if(prob(50))
//33% chance
else
//33% chance

File: code/modules/mob/living/carbon/examine.dm (hunk 4 above)
Comment (Tacolizard) [inline code/modules/mob/living/carbon/examine.dm]: really i only care if each description has an equal chance of appearing so isn't setting them all to 50 fine?

**Comment by Tacolizard** [comment]
really i only care if each description has an equal chance of appearing so isn't setting them all to 50 fine?

**Comment by Cyberboss** [comment]
what you want:

```
if(prob(33))
//33% chance
else if(prob(50))
//33% chance
else
//33% chance
```

**Review by Cyberboss — APPROVED** [review]

File: code/modules/mob/living/carbon/examine.dm [hunk 5]
```diff
@@ -45,26 +45,50 @@
...
+			else
+				msg += "[t_his] body is deformed.\n"
 		else
-			msg += "<b>[t_He] [t_is] severely deformed.</b>\n"
+			if(prob(33))
+				msg += "<B>[t_his] skin is covered in large, unnatural genetic deformities!</B>\n"
+			else if(prob(50))
+				msg += "<B>[t_his] body is covered in lumps and warts!</B>\n"
+			else
+				msg += "<B>[t_his] body and limbs are severely deformed in several areas!</B>\n"
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/examine.dm]: [t_his] is not capitalized and will look very strange here.

Comment (Tacolizard) [inline code/modules/mob/living/carbon/examine.dm]: will [t_His] be capitalized?

**Comment by banthebantz** [comment]
more gore WHEN

**Comment by AnturK** [comment]
I don't like probs in examine, i'm okay with varied descriptions but these should stay the same until you get more beaten up

**Comment by Tacolizard** [comment]
I'm going to delete my fork and start over with new features and ditch github desktop

**Comment by Thunder12345** [comment]
p_his() and similar procs work much better for pronouns
